src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -799,6 +799,19 @@
                 "effect": "deny"
             }
         }'
+  - name: Create a policy definition with mode. The mode 'Indexed' indicates the policy should be evaluated only for resource types that support tags and location. 
```

Comment:
Nice addition here. It would be cool if you could include this explanation of 'Indexed' directly in the parameter help (_params.py), along with explanations for the other values.

Reply:
Adding explanation for all possible modes will result in long help text. Added aka.ms link https://aka.ms/azure-policy-mode for more information.